src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java

```diff
@@ -1011,4 +1011,19 @@ private boolean isVariableMatchesIgnorePattern(String variable) {
         return matcher.matches();
     }
 
+    /**
+     * Check if the statement is zero-distance token.
```

Comment:
Check if the toke should ignored for distance counting .

Reply:
Done.